src/enlyze/schema.py

```diff
@@ -1,17 +1,18 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
Just saw that this line doesn't report as covered. Since this branch is never taken at runtime, it doesn't make sense to consider it for coverage.

```suggestion
if TYPE_CHECKING:  # pragma: no cover
```